backend/app/api/endpoints/deep_research.py

```
@@ -0,0 +1,314 @@
+                async for line in response.aiter_lines():
+                    if line:
+                        yield line + "\n"
+        except httpx.HTTPStatusError as e:
+            logger.error(
+                "[DEEP_RESEARCH] Stream HTTP error: %d - %s",
+                e.response.status_code,
+                e,
+            )
+            yield f"event: error\ndata: {{'error': 'HTTP {e.response.status_code}'}}\n\n"
+        except httpx.RequestError as e:
+            logger.error("[DEEP_RESEARCH] Stream request error: %s", e)
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)